src/v/cluster/scheduling/leader_balancer.h

```diff
@@ -70,6 +72,14 @@ class leader_balancer {
      */
     static constexpr clock_type::duration throttle_reactivation_delay = 5s;
 
+    /*
+     * Used to specify how long a leader should spend trying to recover a
+     * replica the balancer is trying to transfer leadership to.
+     */
```

Comment:
I think this comment is important (and could be fleshed out a bit more), but it should go on the "timeout" member added to the transfer req struct.

This comment could just say "the value for used for transfer timeout in the transfer request".

Reply:
Good point will update this in a subsequent PR.